packages/jest-cli/src/reporters/SummaryReporter.js

```diff
@@ -14,11 +14,11 @@ import type {Context} from 'types/Context';
 import type {ReporterOnStartOptions} from 'types/Reporters';
 import type {PathPattern} from '../SearchSource';
 
-type SummaryReporterOptions = {|
+type SummaryReporterOptions = {
```

Comment:
i'm passing some extra information from `TestRunner`. These options are kind of a hack that was introduced a while ago, since they're kind of a config, but not really. I'll try to refactor it soon, after circus rollout is done